src/py/pyodide/http.py

```diff
@@ -208,7 +208,7 @@ async def unpack_archive(self, *, extract_dir=None, format=None):
 
 
 async def pyfetch(url: str, **kwargs) -> FetchResponse:
-    """Fetch the url and return the response.
+    r"""Fetch the url and return the response.
```

Comment:
FYI, I couldn't run this due to `\*` later, which is not a valid Python escape sequence. I just added an r here, but might be worth checking to see what the normal way to write `**kwargs` in this style of docstring is.

Reply:
It looks like just `**kwargs` is fine and doesn't mess with rst, but could be wrong. https://stackoverflow.com/questions/62511086/how-to-document-kwargs-according-to-numpy-style-docstring

Reply:
I suspect that I added the backslashes to fix some issue with the generated output but I have repeatedly been wrong about this sort of thing so there's a good chance that they are pointless.